packages/editor/patches/@tiptap+extension-list-keymap+2.6.6.patch

```diff
@@ -28,3 +28,13 @@ index f7ab1e4..8645c44 100644
          const $listPos = editor.state.doc.resolve($anchor.before() - 1);
          const listDescendants = [];
          $listPos.node().descendants((node, pos) => {
+@@ -103,6 +105,9 @@ const handleBackspace = (editor, name, parentListTypes) => {
+     if (!isAtStartOfNode(editor.state)) {
+         return false;
+     }
++    if (isNodeActive(editor.state, name) && !hasListItemBefore(name, editor.state)) {
```

Comment:
I don't think the `isNodeActive` check is needed since it is already being checked in the previous lines.

Also, please add a comment that provides info as to why this change is necessary.